src/bus_metrics/aggregation/punctuality_rate.py

```diff
@@ -0,0 +1,126 @@
+"""Class of tools required to reaggregate bus metrics by geographies."""
+import pandas as pd
+import toml
+
+
+class AggregationTool:
+    """Aggregate bus metrics by geographies.
+
+    Parameters
+    ----------
+    stop_level_punctuality: str
+        Full filepath to stop-level punctuality aggregated data
+    config: dict
+        Dictionary of imported toml ingest variables
+    geography: str
+        Geography by which data is to be aggregated
+
+    Attributes
+    ----------
+    code: str
+        Geography code in boundaries data e.g. LSOA21CD
+    name: str
+        Geography name in boundaries data e.g. LSOA21NM
+
+    Methods
+    -------
+    punctuality_by_geography
+        Combine stop-level punctuality data with boundaries
+        and reaggregate
+
+    """
+
+    def __init__(
+        self,
+        stop_level_punctuality: str = "data/stop_level_punctuality/synth_punc.csv",  # noqa: E501
```

Comment:
Hardcoded to synth_punc

Reply:
Resolved to tie in with output from previous PR. Thanks.